optuna/importance/__init__.py

```diff
@@ -3,8 +3,11 @@
 from typing import Optional
 
 from optuna.importance._base import BaseImportanceEvaluator
-from optuna.importance._fanova import FanovaImportanceEvaluator  # NOQA
-from optuna.importance._mean_decrease_impurity import MeanDecreaseImpurityImportanceEvaluator
+from optuna.importance._fanova import FanovaImportanceEvaluator
+
+# fmt: off
+from optuna.importance._mean_decrease_impurity import MeanDecreaseImpurityImportanceEvaluator  # NOQA
+# fmt: on
```

Comment:
How about this?

```suggestion
from optuna.importance._mean_decrease_impurity import (  # NOQA
    MeanDecreaseImpurityImportanceEvaluator,
)
```

refs: https://gitlab.com/pycqa/flake8/-/issues/385

Reply:
It really works... thanks a lot, including the reference. Let me apply this suggestion. @toshihikoyanase please let me know if you have any comments on this.

Reply:
Cool! Thank you for the suggestion.